examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  neutron:
+    template:
+      customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        network_scheduler_driver = neutron.scheduler.dhcp_agent_scheduler.AZAwareWeightScheduler
```

Comment:
this seems specific to dhcp agent

Reply:
This config i found in neutron.conf, however it is disbled, so i will remove this in next commit!